<ui version="4.0" >
 <class>AddWengoAccount</class>
 <widget class="QWidget" name="AddWengoAccount" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>@product@ - Login</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="loginLabel" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>212</red>
           <green>208</green>
           <blue>200</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Light" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Midlight" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>234</red>
           <green>229</green>
           <blue>220</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>128</red>
           <green>128</green>
           <blue>128</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Mid" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>141</red>
           <green>139</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="BrightText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>212</red>
           <green>208</green>
           <blue>200</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Shadow" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>64</red>
           <green>64</green>
           <blue>64</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>10</red>
           <green>36</green>
           <blue>106</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Link" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="LinkVisited" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="AlternateBase" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>232</red>
           <green>232</green>
           <blue>232</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>212</red>
           <green>208</green>
           <blue>200</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Light" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Midlight" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>234</red>
           <green>229</green>
           <blue>220</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>128</red>
           <green>128</green>
           <blue>128</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Mid" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>141</red>
           <green>139</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="BrightText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>212</red>
           <green>208</green>
           <blue>200</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Shadow" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>64</red>
           <green>64</green>
           <blue>64</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>212</red>
           <green>208</green>
           <blue>200</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Link" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="LinkVisited" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="AlternateBase" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>232</red>
           <green>232</green>
           <blue>232</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>106</red>
           <green>104</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>212</red>
           <green>208</green>
           <blue>200</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Light" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Midlight" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>233</red>
           <green>231</green>
           <blue>227</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>128</red>
           <green>128</green>
           <blue>128</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Mid" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>141</red>
           <green>139</green>
           <blue>133</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>106</red>
           <green>104</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="BrightText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>106</red>
           <green>104</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>212</red>
           <green>208</green>
           <blue>200</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>212</red>
           <green>208</green>
           <blue>200</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Shadow" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>10</red>
           <green>36</green>
           <blue>106</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Link" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="LinkVisited" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="AlternateBase" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>232</red>
           <green>232</green>
           <blue>232</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text" >
      <string>&lt;span style="font-size:16pt;">Login&lt;/span>&lt;br>&lt;br>&lt;span>Please enter your email address and&lt;br>password&lt;/span></string>
     </property>
     <property name="indent" >
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="LinkLabel" name="createWengoAccountLabel" >
       <property name="text" >
        <string>Click here if you don't have a @company@ account</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="password" >
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="passwordLabel" >
       <property name="text" >
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="email" />
     </item>
     <item row="2" column="1" >
      <widget class="LinkLabel" name="forgotPasswordLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Forgot your @company@ password?</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="emailAddressLabel" >
       <property name="text" >
        <string>VoxOx Id:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>251</width>
       <height>44</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../../resources/wengophone.qrc" >:/pics/help.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LinkLabel" name="helpLabel" >
       <property name="text" >
        <string>Need help?</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="backButton" >
       <property name="text" >
        <string>&amp;Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton" >
       <property name="text" >
        <string>&amp;Connect</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LinkLabel</class>
   <extends>QLabel</extends>
   <header>qtutil/LinkLabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>email</tabstop>
  <tabstop>password</tabstop>
  <tabstop>backButton</tabstop>
  <tabstop>loginButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../resources/wengophone.qrc" />
  <include location="../../../../resources/wengophone.qrc" />
 </resources>
 <connections/>
</ui>
